### PR TITLE
Removal of hpe-icons in quantum-ux-aspects

### PR DESCRIPTION
**Pull request by donna-marijne**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licenced under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3553

#### Description of Proposed Changes

Several changes have been made in the library to allow `.hpe-icon` and similar implementation-specific selectors to be removed from quantum-ux-aspects.
* Added `IconModule` to the plunker template. Also tidied up indentation.
* In several components, HTML icons have been replaced with `ux-icon` using a unique class to allow targeting in quantum-ux-aspects.
* Dropdown: instead of transforming the icon in the stylesheet, using the correct icon up front.
* Icon Buttons: added `span` so that the CSS margin between icon and text applies.
* Updated .gitignore for the ux-icons font files.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-3553-remove-hpe-icon

### CODE REVIEW

**Comment by ashley-hunter** [comment]
Some tests appear to be failing now that were previously targeting `.hpe-icon` elements

**Comment by ashley-hunter** [comment]
Fixed failing tests

**Comment by donna-marijne** [comment]
Thanks for that, didn't re-check for the additional number picker changes.

File: docs/app/pages/css/css-sections/buttons/icon-buttons/icon-buttons.component.html [hunk 1]
```diff
@@ -60,22 +60,22 @@ <h4>Icon and Text</h4>
         <div class="btn-container m-b-md">
 
             <button type="button" class="btn button-primary">
-                <span class="hpe-icon hpe-add" aria-hidden="true"></span>&nbsp;Add
+                <span class="hpe-icon hpe-add" aria-hidden="true"></span><span>Add</span>
```
[the remark this answers is not in the document]
Comment (donna-marijne) [reply docs/app/pages/css/css-sections/buttons/icon-buttons/icon-buttons.component.html]: Thanks you for catching this! Pushing a fix now.

**Pull request merged**